regolith/helpers/a_proposalhelper.py

```diff
@@ -128,6 +131,12 @@ def db_updater(self):
                 end_date = date_parser.parse(rc.end_date).date()
                 if begin_date > end_date:
                     raise ValueError("begin_date can not be after end_date")
+                if rc.duration:
+                    expected_duration = 12*relativedelta(end_date, begin_date).years + relativedelta(end_date, begin_date).months
```

Comment:
how will this behave if there is not an integral number of months?

Reply:
`relativedelta(date1, date2).months` returns the floor of the difference in months, whereas `begin_date + relativedelta(months=rc.duration)`only supports integral values for `month` as defined in the documentation. so in this equality, I should probably add a floor(float(rc.duration))... unless we want to be comparing exact differences? i.e. the difference between `begin_date` and `end_date` should also be a float?

Reply:
not sure I have a preference as long as it works.  Maybe think about some edge cases, how you would want them to behave, and then how they do behave given the different options.  This is where TDD is helpful because we test for the edge cases, then make the code work.  One way to do this is if we break this out into a function that we put into dates.py.  It may make sense or it may not.....